src/main/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommand.java

```diff
@@ -84,7 +84,8 @@ public CommandResult execute(Model model) throws CommandException {
         OrderList orderList = model.getSelectedOrderList();
         String feedbackToUser = !orderToEdit.isSameOrder(editedOrder)
                 && orderList.contains(editedOrder)
-                ? MESSAGE_DUPLICATE_ORDER_WARNING
+                ? String.format(MESSAGE_DUPLICATE_ORDER_WARNING,
+                editedOrder.getStatus().toString().toLowerCase())
```

Comment:
![image](https://github.com/user-attachments/assets/84b20757-589a-4921-9be5-8257a8f37040)
kx made this method is previous commit.